msticnb/nb/azsent/network/ip_summary.yaml

```diff
@@ -35,6 +35,7 @@ output:
   get_az_netflow:
     title: Azure network analytics netflow data for IP.
     text:
+      (only available for if Azure network analytics net flow enabled.)
```

Comment:
I think the official name is Azure NSG Flow Logs - might be useful to have that to guide people as searching 'Azure network analytics net flow' doesn't return anything useful

Reply:
Tis fixed